4-4-1.py

```diff
@@ -0,0 +1,53 @@
+'''
+
+Вывести матрицу 1
+
+На вход программе подаются два натуральных числа nnn и mmm, каждое на отдельной строке — количество строк и столбцов в матрице. Далее вводятся сами элементы матрицы — слова, каждое на отдельной строке; подряд идут элементы сначала первой строки, затем второй, и т.д.
+
+Напишите программу, которая сначала считывает элементы матрицы один за другим, затем выводит их в виде матрицы.
+
+Формат входных данных
+На вход программе подаются два числа nnn и mmm — количество строк и столбцов в матрице, далее идут n×mn \times mn×m слов, каждое на отдельной строке.
+
+Формат выходных данных
+Программа должна вывести считанную матрицу, разделяя ее элементы одним пробелом.
+Тестовые данные 🟢
+
+Sample Input 1:
+
+4
+2
+и
+швец
+и
+жнец
+и
+на
+дуде
+игрец
+
+Sample Output 1:
+
+и швец
+и жнец
+и на
+дуде игрец
+'''
+
+def print_matrix(row, col):
```

Comment:
`row` and `col` are too abstract and can mean many things, I would name them something like`row_count`, `col_count` to make it clear that they contain the count of rows and cols.